docs/book/content/types/unions.md

```diff
@@ -187,3 +188,34 @@ impl Character {
 
 # fn main() {}
 ```
+
+## Enums (Derive)
+
+This example is similar to `Enums (Impl)`. To successfully use the
+derive macro, ensure that each variant of the enum has a different
+type. Since each variant is different, the device macro provides
+`std::convert::Into<T>` converter for each _used_ variant. Fields
+which are skipped by using `#[graphql(skip)]` do not have a
```

Comment:
This is no longer true, correct? It appears below you explicitly forbid skip for this (which feels right).